src/StoreApi/Routes/CartUpdateCustomer.php

```diff
@@ -170,17 +172,10 @@ protected function maybe_update_order() {
 				'shipping_state'      => wc()->customer->get_shipping_state(),
 				'shipping_postcode'   => wc()->customer->get_shipping_postcode(),
 				'shipping_country'    => wc()->customer->get_shipping_country(),
+				'shipping_phone'      => wc()->customer->get_shipping_phone(),
```

Comment:
This is supported with WC 5.6 +

Reply:
Moved to https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/5326